cetmix_tower_server/tests/__init__.py

```diff
@@ -1,19 +1,19 @@
-from . import test_server
+# from . import test_server
 from . import test_command
-from . import test_file
-from . import test_file_template
-from . import test_plan
-from . import test_plan_line
-from . import test_plan_line_action
-from . import test_command_log
-from . import test_plan_log
-from . import test_server_log
-from . import test_server_template
-from . import test_variable
-from . import test_variable_value
-from . import test_variable_option
-from . import test_command_wizard
-from . import test_reference_mixin
-from . import test_update_related_variable_names
-from . import test_key
-from . import test_cetmix_tower
+# from . import test_file
+# from . import test_file_template
+# from . import test_plan
+# from . import test_plan_line
+# from . import test_plan_line_action
+# from . import test_command_log
+# from . import test_plan_log
+# from . import test_server_log
+# from . import test_server_template
+# from . import test_variable
+# from . import test_variable_value
+# from . import test_variable_option
+# from . import test_command_wizard
+# from . import test_reference_mixin
+# from . import test_update_related_variable_names
+# from . import test_key
+# from . import test_cetmix_tower
```

Comment:
why are you comment all?

Reply:
> why are you comment all?

My bad, forgot to uncomment it. Thank you for noticing this!